.github/workflows/github-actions-tests.yml

```diff
@@ -1,5 +1,7 @@
 name: Tests
-on: [ pull_request ]
+on:
+  pull_request:
+  push:
```

Comment:
I'm not so sure about this seemingly unrelated change.
Won't this run multiple jobs for a "PR open" and a "branch push"?

Reply:
Hi thanks to check this PR.
I used it in for CI tests on current branch before make the PR. That part has been removed now.